Sources/Controllers/2.0/BottomSheet/OABaseBottomSheetViewController.xib

```diff
@@ -127,14 +127,16 @@
                             </subviews>
                             <color key="backgroundColor" red="0.94901960780000005" green="0.94901960780000005" blue="0.95294117649999999" alpha="1" colorSpace="calibratedRGB"/>
                         </view>
-                        <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Wg5-91-0hA">
+                        <view contentMode="scaleToFill" id="Wg5-91-0hA">
                             <rect key="frame" x="0.0" y="448" width="414" height="94"/>
+                            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
```

Comment:
Are any of the changes in this xib are necessary? Maybe you should revert all of them